Gemfile

```diff
@@ -36,4 +36,5 @@ gem "webpacker", "~> 5.0"
 gem "bootsnap", ">= 1.4.2", require: false
 
 gem "view_component", path: ENV["VIEW_COMPONENT_PATH"] if ENV["VIEW_COMPONENT_PATH"]
-gem "view_component_storybook", "~> 0.8.0"
+# gem "view_component_storybook", "~> 0.10.1"
```

Comment:
Will use a proper released version once the referenced branch below is incorporated into `vioew_component_storybook`.